Design teams set up to address antitrust concerns

AramZS opened the pull request:
Intended to address: concerns that the charter doesn't fully comply with the W3C Antitrust and Competition Guidance. Design teams are used by the WG to lead the creation of specifications and features, adding this text is intended to assure that these teams are constructed with the antitrust concerns in mind.

See https://github.com/patcg/patwg-charter/issues/44 to help with understanding this issue.

martinthomson commented on charter.html:
```diff
@@ -374,6 +374,7 @@ <h2>
         <p>
           At the discretion of the Chairs and subject to a consensus call, the Working Group may designate a Design Team to study a particular subject, and report back to the Working Group their findings and a recommendation. These Design Teams shall consist of a small group of volunteers who commit to investing significant amounts of time on the task over a short, time-bound, period.
         </p>
+        <p>When volunteers are available the chairs will endeavor to make such Design Teams filled with a diverse set of stakeholders including participants at multiple levels of the ad ecosystem</p>
```
```suggestion
        <p>When volunteers are available the chairs will endeavor to make such Design Teams filled with a diverse set of stakeholders including participants at multiple levels of the ad ecosystem.</p>
```

mnot commented:
This is a significant change from how design teams function elsewhere, both inside and outside the IETF. 

The point of a design team is for it to iterate quickly on making a proposal to the full group, which the group can then accept or deny; the design team's output is given no weight over any other proposal or party. By requiring broad participation in the team, this effectively makes it a subcommittee, robbing it of its efficiency but still not ensuring success.

It's the worst of both worlds, and it also sets a concerning precedent in chartering. So I'm -1 on this.

AramZS commented:
@mnot Have other groups both inside and outside the W3C addressed concerns about collusion or the dominance of a particular organization among the set of individual contributors in ways that it might be useful to reference in helping us to resolve the issues raised by some of our participants?

martinthomson commented:
That sort of bias is something that can be left to chairs - and the working group - to manage.

Let's say that companies X and Y are strongly dominant in some way.  A design team is formed, with a large contingent from those companies.  The output of that design team strongly favours these companies.  The working group - which should be more diverse - can reject that output on that basis.

Much of this is inherent in the W3C [decision-making process](https://www.w3.org/2021/Process-20211102/#Consensus).  The IETF follows a [similar](https://datatracker.ietf.org/doc/html/rfc2026#section-6.5) [process](https://datatracker.ietf.org/doc/html/rfc2418#section-3.4), following similar [principles](https://datatracker.ietf.org/doc/html/rfc3935).

If the concern is about anti-trust policies, I'm less familiar with those.

mnot commented:
As far as I'm aware the [antitrust and competition guidance](https://www.w3.org/Consortium/Legal/2017/antitrust-guidance) has no bearing on this issue at all; it places requirements on participants, not WGs, and affirms that the Process is designed to support 'fair and open deliberations.' As long as the charter conforms to the process, it should be fine.

AramZS commented:
I am relaying a comment from a member who has raised formal objections. I do not intend to imply support or opposition to this statement, but just to make their objection on this available for discussion with the member's permission: 

> A decision making process is needed which is clear and fair. Voting was rejected. Perhaps this can be revisited?
